test/dataframe.jl

```diff
@@ -2187,7 +2188,7 @@ end
                     @test_throws DimensionMismatch("Permutation does not have a correct length ($len != $perm_len)") permute!(df, p)
                     @test_throws DimensionMismatch("Permutation does not have a correct length ($len != $perm_len)") invpermute!(df, p)
                 elseif sort(p) != collect(1:perm_len)
-                    if perm_len <= 4 || rand() < .005
+                    if perm_len <= 5 || rand() < .01
```

Comment:
maybe just check it always, i.e. remove the `rand() < 0.01` (what is the benefit of keeping this additional condition?)

Reply:
The benefit is runtime. Without the check runtime jumps from 6 seconds to 143 seconds.

Reply:
40,661

Reply:
OK - this is enough :)